sarathi/sarathi.py

```diff
@@ -1,7 +1,7 @@
 """Sarathi - A discord bot to steer through the battlefield of knowledge"""
 import os
 import sys
-import discord
+import discordcd
```

Comment:
Please be careful when committing code, you seem to have typed `cd` into the editor by mistake and you saved it. Change this back.